### PR TITLE
yoav  - Neo roller 

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

**Review by aaa-png — APPROVED** [review]
xing xong

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 1]
```diff
@@ -80,15 +85,17 @@ public SmartJoystick getFourthJoystick() {
...
-        mainJoystick.A.whileTrue(new GoToShootingState());
+//        secondJoystick.START.whileTrue(new PanicMode());
+//        secondJoystick.BACK.whileTrue(new ToggleScoringMode());
+//        secondJoystick.A.whileTrue(new MoveElbowAndWrist(ElbowConstants.PresetPositions.SCORE.ANGLE, WristConstants.PresetPositions.SCORE.ANGLE));
+//        secondJoystick.B.whileTrue(new MoveElbowAndWristToSafe());
+//        secondJoystick.X.whileTrue(new MoveShooterToAngle(PivotConstants.PresetPositions.PICK_UP.ANGLE));
+//        secondJoystick.Y.whileTrue(new MoveShooterToAngle(PivotConstants.PresetPositions.TRANSFER.ANGLE));
+//        secondJoystick.POV_DOWN.whileTrue(new CollectNoteToScoringMode());
+//
+//        mainJoystick.A.whileTrue(new GoToShootingState());
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/OI.java:96]: no

File: src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunByPower/RunRollerClockwiseUntilObjectIsOut.java [hunk 2]
```diff
@@ -1,6 +1,8 @@
-package edu.greenblitz.robotName.commands.arm.roller;
+package edu.greenblitz.robotName.commands.arm.roller.RunByPower;
 
-public class RunRollerClockwiseUntilObjectIsOut extends RollerCommand{
+import edu.greenblitz.robotName.commands.arm.roller.RollerCommand;
+
+public class RunRollerClockwiseUntilObjectIsOut extends RollerCommand {
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunByPower/RunRollerClockwiseUntilObjectIsOut.java:5]: dont you need an end that stops the roller?

File: src/main/java/edu/greenblitz/robotName/commands/getNoteToSystem/MoveNoteBetweenShooterAndArmWithSafeRotations.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package edu.greenblitz.robotName.commands.getNoteToSystem;
+
+import edu.greenblitz.robotName.ScoringModeSelector;
+import edu.greenblitz.robotName.commands.arm.roller.MoveNoteToMiddleOfRoller;
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class MoveNoteBetweenShooterAndArmWithSafeRotations extends ConditionalCommand {
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/commands/getNoteToSystem/MoveNoteBetweenShooterAndArmWithSafeRotations.java:8]:
```suggestion
public class MoveNoteBetweenShooterAndArm extends ConditionalCommand {
```

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/roller/RollerInputs.java [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
 public class RollerInputs {
-	
+
 	public double appliedOutput;
-	
+
 	public double outputCurrent;
 
-	public boolean isObjectInArm;
+	public double temperature;
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/roller/RollerInputs.java:13]: no

**Review by tnrz135 — APPROVED** [review]

**Pull request merged**